lib/app_server.py

```diff
@@ -28,6 +28,10 @@ def run_server(execs, cwd, server, logfile, retval):
 class AppTest(Test):
     def execute(self, server):
         server.current_test = self
+        if self.suite_ini.get('pre_cleanup', False):
+            for regx in server.re_vardir_checkpoints:
+                for f in glob.glob(os.path.join(server.vardir, regx)):
+                    os.remove(f)
```

Comment:
Please, move it to Server of Tarantool/AppServer fucncition(s) with or without the condition.